frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:tripmeout/widgets/map_widget.dart';
+
+//TODO: Add loading screen after to the Create Trip Widget
+//TODO: Get rid of the newInformation text on the page.
+
+class CreateTripWidget extends StatefulWidget {
+  @override
+  _CreateTripWidgetState createState() => _CreateTripWidgetState();
+}
+
+class _CreateTripWidgetState extends State<CreateTripWidget> {
+  String place = "No Input";
+  int radius = 0;
+  String newInformation = 'Grabbed info placed here.';
+  changeText() {
+    setState(() {
+      newInformation = place + " " + radius.toString() + " km";
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return FittedBox(
+      child: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.all(25.0),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Container(
+                  width: 250.0,
+                  child: TextFormField(
+                    onChanged: (text) {
+                      place = text;
+                    },
+                    textAlign: TextAlign.center,
+                    decoration: InputDecoration(
+                      border: OutlineInputBorder(),
+                      labelText: 'Enter your Location',
+                    ),
+                  ),
+                ),
+                Container(
+                  width: 125.0,
+                  child: TextFormField(
+                    onChanged: (text) {
+                      radius = int.parse(text);
+                    },
+                    textAlign: TextAlign.center,
+                    keyboardType: TextInputType.number,
+                    inputFormatters: <TextInputFormatter>[
+                      WhitelistingTextInputFormatter.digitsOnly
+                    ],
+                    decoration: InputDecoration(
+                        border: OutlineInputBorder(), labelText: 'Radius KM'),
+                  ),
+                ),
+              ],
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.all(25.0),
+            child: MapWidget(),
+          ),
+          Container(
+              child: Text('$newInformation', style: TextStyle(fontSize: 21))),
```

Comment:
Try to never create your text style inside of a widget. If you still want to change this to not look like default text you can get the ThemeData from the context, and use that to get things like header styles. This enables your app to look similar everywhere and you only need to change the ThemeData to update stuff.

For something similar to what you have, you probably want something like this:
```dart
Text('$newInformation', style: Theme.of(context).textTheme.headline5)
```